src/styles/sc-global.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```

Comment:
What is the purpose of this file inside a new styles directory? Wouldn't it be more consistent to create a new component inside `components/Link/Link.js`?

Reply:
I was completely overthinking that. Thank you. I was stuck for a long time thinking about that heh.

Reply:
Yeah, I know that feeling :-)